projects/plugins/crm/admin/dashboard/dashboard.ajax.php

```diff
@@ -89,19 +92,20 @@ function jetpackcrm_dash_refresh() {
 		$zeros['year'][ $v->year ] = $v->count;
 	}
 
-	// convert the monthly array into a zero padded one
 	foreach ( $monthly as $v ) {
-		$the_month                    = gmdate( 'M y', $v->ts );
+		$datetime_month               = DateTime::createFromFormat( '!m Y', $v->month . ' ' . $v->year );
+		$the_month                    = $datetime_month->format( 'M y' );
```

Comment:
I still think the timestamp way is more readable than this way, though that'd require keeping that column in the database.